src/commands/dev/index.js

```diff
@@ -35,47 +34,59 @@ const startStaticServer = async ({ settings, log }) => {
   log(`\n${NETLIFYDEVLOG} Server listening to`, settings.frameworkPort)
 }
 
+const NOT_FOUND_EXIT_CODE = 127
+
+const isNonExistingCommandError = (error) => {
+  // `ENOENT` is only returned for non Windows systems
+  // See https://github.com/sindresorhus/execa/pull/447
+  // exitCode === 127 is returned from npm/yarn
```

Comment:
I might be wrong, but I think exit codes are program-specific, not OS-specific.
`127` is specific by Bash when calling a non-existing command (including npm or yarn).
The reason why Windows appears to behave differently is because CI tests use `cmd.exe` instead of Bash on Windows.

That being said, the best we can do to detect this is basically what this function is doing, so this looks good :+1: